docs/src/whatsnew/latest.rst

```diff
@@ -30,29 +30,6 @@ This document explains the changes made to Iris for this release
 ✨ Features
 ===========
 
-#. `@bsherratt`_ added support for plugins - see the corresponding
-   :ref:`documentation page<community_plugins>` for further information.
-   (:pull:`5144`)
-
-#. `@rcomer`_ enabled lazy evaluation of :obj:`~iris.analysis.RMS` calcuations
-   with weights. (:pull:`5017`)
-
-#. `@schlunma`_ allowed the usage of cubes, coordinates, cell measures, or
-   ancillary variables as weights for cube aggregations
-   (:meth:`iris.cube.Cube.collapsed`, :meth:`iris.cube.Cube.aggregated_by`, and
-   :meth:`iris.cube.Cube.rolling_window`). This automatically adapts cube units
-   if necessary. (:pull:`5084`)
-
-#. `@lbdreyer`_ and `@trexfeathers`_ (reviewer)  added :func:`iris.plot.hist` 
-   and :func:`iris.quickplot.hist`. (:pull:`5189`)
-
-#. `@tinyendian`_ edited :func:`~iris.analysis.cartography.rotate_winds` to
-   enable lazy computation of rotated wind vector components (:issue:`4934`,
-   :pull:`4972`)
-
-#. `@ESadek-MO`_ updated to the latest CF Standard Names Table v80
-   (07 February 2023). (:pull:`5244`)
-
```

Comment:
These are all v3.5 entries, inadvertently re-added to the whatsnew during #5191 rebase shenanegans.

Reply:
@rcomer Great spot!